src/controllers/user-controller.js

```diff
@@ -0,0 +1,70 @@
+const userService = require("../service/user-service");
```

Comment:
userService is a class, so the name of the variable will be according to PascalCase